lib/utils/import/simplenote/test.ts

```diff
@@ -42,6 +50,90 @@ describe('SimplenoteImporter', () => {
     });
   });
 
+  describe('ZIP file import', () => {
+    it('should process ZIP files by calling processZipFile method', () => {
+      const zipFile = new File(['content'], 'test.zip', {
+        type: 'application/zip',
+      });
+      importer.processZipFile = jest.fn();
+
+      importer.importNotes([zipFile]);
+
+      expect(importer.processZipFile).toHaveBeenCalledWith(zipFile);
+    });
+
+    it('should emit error when ZIP file is empty in processZipFile', () => {
+      const zipFile = new File([''], 'test.zip', { type: 'application/zip' });
+      const mockFileReader = new FileReader();
+      (FileReader as any).mockImplementation(() => mockFileReader);
+
+      importer.processZipFile(zipFile);
```

Comment:
this could be calling `importer.importNotes()` as in the previous test, but as it is, it’s reaching into an internal method making the test brittle and at risk of diverging from the actual app code